src/software/ai/motion_constraint/motion_constraint_visitor.cpp

```diff
@@ -5,8 +5,8 @@
 void MotionConstraintVisitor::visit(const GoalieTactic &tactic)
 {
     current_motion_constraints.erase(TbotsProto::MotionConstraint::FRIENDLY_DEFENSE_AREA);
-    current_motion_constraints.erase(
-        TbotsProto::MotionConstraint::HALF_METER_AROUND_BALL);
+//    current_motion_constraints.erase(
+//        TbotsProto::MotionConstraint::HALF_METER_AROUND_BALL);
```

Comment:
accidental change, revert

Reply:
pretty sure it needs to go - causes goalie to chip during stop play

Reply:
nevermind it should probably stay - stop play is a weird edge case we probably wont hit